src/content/components/IdleSearchField.js

```diff
@@ -6,42 +6,35 @@ import './IdleSearchField.css';
 class IdleSearchField extends Component {
```

Comment:
In the future it would be nice if you're making substantial changes to a component to go ahead and add flow typing to it.